### DISCUSSION TITLE
Kubernetes Dashboard

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#133] [Ideas]
It would be great to see Kubernetes Dashboard as an extension. I would like to see visually what is running on my local machine when I am developing applications. The Kubernetes dashboard extension can also install the dashboard first checking for requirements. See this link for installing the Kubernetes Dashboard (https://gist.github.com/deeja/c63ebfc7dac0fdf3970bbca1185d3c43).

Copied over from prior submission form.

**Comment by sandipchitale** [comment]
https://github.com/sandipchitale/kubernetes-dashboard

**Reply by Anxhul10** [reply]
Really nice repo, thanks for sharing! 🙌
I'm currently trying to build a Docker Extension myself and found this idea + repo very insightful.
Would appreciate any tips or best practices you followed while building this project!
